recipe/meta.yaml

```diff
@@ -96,18 +101,21 @@ outputs:
       string: cpu_0
 
   - name: r-xgboost
-    script: install-r-xgboost.sh
+    script: install-r-xgboost.sh  # [not win]
+    script: install-win-wrapper.bat  # [win]
```

Comment:
You could probably do a much simpler script for windows:

```
@echo on
pushd %SRC_DIR%\R-package
%R% CMD INSTALL --preclean --build .
if errorlevel 1 exit 1
```

Then you don't need a wrapper and this would then also activate the build environment correctly.